pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -919,6 +919,14 @@ func (p *PriorityQueue) Update(logger klog.Logger, oldPod, newPod *v1.Pod) error
 	p.lock.Lock()
 	defer p.lock.Unlock()
 
+	if p.isSchedulingQueueHintEnabled {
```

Comment:
@wackxu @sanposhiho 

Hi, I'm a bit puzzled, why do we need to implement in-flight-pods check within the method `PriorityQueue.Update()`? 

Is it more appropriate to impl it in `updatePodInSchedulingQueue`?
https://github.com/kubernetes/kubernetes/blob/c6064489223862fe1888fcbe0656ab1087461adb/pkg/scheduler/eventhandlers.go#L134-L155

Reply:
`inFlightPods` isn't exposed, so we can't impl it in `updatePodInSchedulingQueue`, unless we add to new function to SchedulingQueue interface.

Reply:
ok~ by the way, if this MR get merged, then the `assumed` check  in `updatePodInSchedulingQueue` can be removed?

Reply:
oooh,

> ok~ by the way, if this MR get merged, then the `assumed` check in `updatePodInSchedulingQueue` can be removed?

ooh, it should not be removed, pls ignore it. Pods will exist in `assumedPods` and be removed from`inFlightPods` when it completes the bind plugins.